openquake/hazardlib/gsim/mgmpe/m9_basin_term.py

```diff
@@ -0,0 +1,79 @@
+# The Hazard Library
+# Copyright (C) 2012-2023 GEM Foundation
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module :mod:`openquake.hazardlib.mgmpe.m9_basin_term` implements
+:class:`~openquake.hazardlib.mgmpe.M9BasinTerm`
+"""
+import numpy as np
+
+from openquake.hazardlib import const
+from openquake.hazardlib.gsim.base import GMPE, registry
+
+
+def _get_m9_basin_term(ctx, imt, mean):
+    if imt.period > 1.9: # Only apply to long-period SA
```

Comment:
Since this functions works by side-effect by modifying the mean, a better name would be `_apply_m9_basin_term`.

Reply:
renamed this function